community/tpm2-tss/APKBUILD

```diff
@@ -1,27 +1,86 @@
-# Contributor: Alexander Sack <asac@pantacor.com>
+# Maintainer: Erik Larsson <who+alpine@cnackers.org>
 # Maintainer: Alexander Sack <asac@pantacor.com>
-pkgname="tpm2-tss"
-pkgver="2.3.1"
+# Contributor: Erik Larsson <who+alpine@cnackers.org>
+# Contributor: Alexander Sack <asac@pantacor.com>
```

Comment:
Please just have one Maintainer and have it at the last line of this comment block (so the `Contributor` lines are above it).